chisel.lua

```diff
@@ -91,24 +87,23 @@ local function chisel_interact(player, pointed_thing, is_right_click)
 	else
 		if is_sneak then
 			-- Backward cycle mode
-			for k,v in pairs(group) do
+			for _,v in pairs(group) do
 				if v == node_name then break end
-				new_style = k
 				new_node_name = v
 			end
 
 			if new_node_name == nil then
 				-- Not found? Go for the last element
-				for k,v in pairs(group) do
-					new_style = k
+				for _,v in pairs(group) do
 					new_node_name = v
 				end
 			end
 		else
 			-- Forward cycle mode
-			new_style , new_node_name = next(group,style)
+			local _
```

Comment:
This is a hack, any ideas to improve it?

Reply:
You can use `select`:
```lua
new_node_name = select(2, next(group,style))
```